flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/connector/writer/KuduWriterConfig.java

```diff
@@ -65,6 +101,16 @@ public String toString() {
     public static class Builder {
         private String masters;
         private FlushMode flushMode = FlushMode.AUTO_FLUSH_BACKGROUND;
+        // Reference from AsyncKuduClientBuilder defaultOperationTimeoutMs.
+        private long timeout = 30000;
```

Comment:
could we use AsyncKuduClientBuilder.DEFAULT_OPERATION_TIMEOUT_MS directly?

private long timeout = AsyncKuduClientBuilder.DEFAULT_OPERATION_TIMEOUT_MS;

Reply:
Sure,  I have update the code,  It seems that session-related parameters cannot be directly referenced.